src/ai/backend/agent/server.py

```diff
@@ -494,12 +496,14 @@ async def execute(
     @collect_error
     async def execute_batch(
         self,
+        session_id: str,
```

Comment:
This RPC function is unused and can be removed.

Reply:
Thanks for the feedback! This function has been removed by commit d9bb407.